src/models/Settings.js

```diff
@@ -135,8 +135,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'customButtons': ['array', false, []],
 
       //Registration
-      'registration': 'object',
-      'registration.click': 'function'
+      'registration.enable': ['boolean', false, false],
```

Comment:
why do we need this enabled flag ? And what is the benefit of adding it here instead of adding it to  features object ?